profiler/src/ProfilerEngine/Datadog.Profiler.Native/RawSample.h

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include <stdint.h>
+#include <vector>
+#include "cor.h"
+#include "corprof.h"
+#include "ManagedThreadInfo.h"
+
+
+class RawSample
```

Comment:
Since it's a base class, you should add a virtual destructor.

Reply:
✅